### DISCUSSION TITLE
Bootstrap 5

### DISCUSSION THREAD

**Discussion by helgator** [selectize/selectize.js#1978] [General]
Hi, I am new to selectize. I was able to get it working but how do I enable the Bootstrap 5 styles?

Is there a way to make it work with Bootstrap 5 "floating labels"?

Thanks

**Comment by JDCain** [comment]
@helgator Did you ever find a solution for this? I'm looking for one as well. As far as I can see it's not supported in the bs5 theme scss.

**Comment by jlwegaba** [comment]
> Hi, I am new to selectize. I was able to get it working but how do I enable the Bootstrap 5 styles?
> 
> Is there a way to make it work with Bootstrap 5 "floating labels"?
> 
> Thanks


The quickest way to enable BOOTSTRAP 5 working with your selectize is to just  use **selectize.bootstrap5.css** . 

Download the file here [https://github.com/selectize/selectize.js/blob/master/dist/css/selectize.bootstrap5.css](selectize.bootstrap5.css)

Then remove the default selectize css file - **selectize.default.min.css**

**Reply by JDCain** [reply]
I had tried to grab the scss previously but there was no styling for the floating labels in bootstrap 5.

**Reply by KES777** [reply]
Should I remove 'selectize.css' also?

**Comment by nandocunha** [comment]
I use bootstrap's dark and light option and it can be changed dynamically, how is it possible to change the selectize style in this case?

**Comment by Infomaquina** [comment]
.selectize-input {
   padding-top: 26px !important; 
   padding-bottom: 6px !important;
   border-top-left-radius: 5px !important;
   border-bottom-left-radius: 5px !important;
}
